ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard.schema.yml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_postcard:
+    type: dict
+    keys:
+      disabled:
+        type: bool
+        description: Enable or disable the postcard telemetry feature
+      ingress:
+        type: dict
+        description: Configure ingress parameters
+        keys:
+          collection:
+            type: dict
+            description: Collector configuration
+            keys:
+              source:
+                type: str
+                description: GRE source configuration
```

Comment:
These descriptions are not clear. A user needs to understand what is expected here. Do these fields expect an IP address or an interface or ?.

Reply:
Added detailed description.